client/app/vps/import/vps.service.js

```diff
@@ -807,14 +807,31 @@ angular.module("managerApp").service("VpsService", [
             return $http.get(["apiv6/order/vps/", vps.name, "/snapshot"].join(""));
         };
 
+        this.getOptionSnapshotFormated = function (vps) {
+            if (vps.version === "_2015_V_1" && vps.offerType === "SSD") {
+                return this.getOptionDetails(vps).then(d => {
```

Comment:
chain then instead of nesting them in one another

Reply:
Yes done.